### ISSUE TITLE
Tags missing for v1.0 RC releases sometimes

### ISSUE THREAD

**Issue by stefreak** [kubernetes-client/javascript#1828]
**Describe the bug**
v1.0 RC releases sometimes are missing git tags sometimes.

Only [1.0.0-rc5](https://github.com/kubernetes-client/javascript/releases/tag/1.0.0-rc5) and [1.0.0-rc3](https://github.com/kubernetes-client/javascript/releases/tag/1.0.0-rc3) have been pushed to the repository.

**Additional context**
Tags (And GitHub releases) can help quickly parsing the list of changes when troubleshooting an issue, or when making a decision to update.

Happy to share that we are successfully using the release-1 branch at [github.com/garden-io/garden](@garden-io/garden) since a while and are satisfied with it, only very rarely seeing bugs related to the client.
If you need hands let us know

**Comment by brendandburns** [comment]
Yeah, the trouble comes when I had to run the release manually instead of the github action. We should be in an ok place now.

Thanks also for the report that it is working for you. I think we're moving towards a 1.0.0 release associated with the next Kubernetes release.

**Comment by cjihrig** [comment]
@brendandburns it looks like this issue was still present on the 1.0.0 release - https://github.com/kubernetes-client/javascript/actions/runs/12395838409/job/34602484401.
